server/src/main/java/org/elasticsearch/search/aggregations/support/CoreValuesSourceType.java

```diff
@@ -288,4 +291,6 @@ public String value() {
         return name().toLowerCase(Locale.ROOT);
     }
 
+    /** List containing all members of the enumeration. */
+    public static List<ValuesSourceType> ALL = Arrays.asList(CoreValuesSourceType.values());
```

Comment:
I think `ALL` might be a bit confusing here. I wonder if renaming it into `ALL_CORE` or using `Arrays.asList(CoreValuesSourceType.values())` instead of the constant will better demonstrate the actual scope here for future readers.

Reply:
To me it seemed clear that `CoreValuesSourceType.ALL` is about all core VST. I renamed it to `CoreValuesSourceType.ALL_CORE` hoping that this can't be misunderstood.